themes/digital.gov/src/scss/new/_topics-button.scss

```diff
@@ -0,0 +1,88 @@
+@use "uswds-core" as *;
+
+.topics-button {
+  margin-bottom: units("105");
+
+  &__link {
+    @include u-font("sans", "sm");
+    @include u-text("medium");
+    @include u-text("no-underline");
+    @include u-bg("white");
+    @include u-text("primary-vivid");
+    @include u-border("1px", "primary-vivid", "solid");
+    @include u-radius("sm");
+    display: block;
+    overflow: auto;
+    padding: units(1);
+    transition: background-color 0.5s, color 0.5s;
+    white-space: nowrap;
+
+    @include at-media("tablet") {
+      padding-left: units(1);
+    }
+  }
+
+  &__arrow {
+    @include u-float("right");
+    @include u-text("gray-1");
+    display: none;
+    margin-right: 10px;
+    transition: margin 0.4s;
+
+    @include at-media("tablet-lg") {
+      display: block;
+    }
+  }
+
+  &__list {
+    list-style: none;
+    padding-left: 0;
+
+    @include at-media("tablet-lg") {
+      column-count: 3;
+    }
+  }
+
+  &__number {
+    @include u-padding-x("05");
+    @include u-radius("pill");
+    @include u-bg("primary-lightest");
+    @include u-text("base-darker");
+    @include u-font("sans", "2xs");
+    height: fit-content;
+    margin: auto;
+  }
+}
+
+.topics-button--primary {
+  .topics-button__link {
+    &:hover {
+      @include u-bg("primary-vivid");
+      @include u-text("gray-1");
+      @include u-border("1px", "primary-vivid", "solid");
+      span {
```

Comment:
This should be a class, not an HTML element.

Not a big deal here, but we should try to stick to 3 levels deep of nesting.

Reply:
updated the span html element to the primary variant to the span that is class `arrow`